config/kcp-rbac/role.yaml

```diff
@@ -50,4 +50,4 @@ rules:
   resources:
   - apiexports/content
   verbs:
-  - update
+  - *
```

Comment:
must be `'*'` otherwise kustomize failing

Reply:
Fixed. Thanks.